README.md

```diff
@@ -61,32 +61,60 @@ $ make build
 $ ./bin/cluster-autoscaler-operator --kubeconfig=<PATH/TO/YOUR/CLUSTERS/KUBECONFIG> ...
 ```
 
-### Build the e2e tests
+### Run the autoscaler e2e tests
 
-```console
-$ make build-e2e 
-go test -c -o bin/e2e github.com/openshift/cluster-api-actuator-pkg/pkg/e2e
-```
+Before running the autoscaler e2e tests, ensure that you have exported the
+`KUBECONFIG` environment variable, or that you have a valid config in
+`$HOME/.kube/config`. By default, these tests will run in the current namespace.
 
-### Run the autoscaler e2e tests
+These tests use the [Ginkgo](https://onsi.github.io/ginkgo/) framework, you
+can pass command line options directly through the `ci-integration.sh` script.
+For example, use `-focus` to restrict the tests you are running.
 
 ```console
-$ NAMESPACE=kube-system ./hack/ci-integration.sh -ginkgo.focus "Autoscaler should" -ginkgo.v -ginkgo.dryRun
-=== RUN   TestE2E
-Running Suite: Machine Suite
-============================
-Random Seed: 1562320813
-Will run 1 of 15 specs
-
-[Feature:Machines][Serial] Autoscaler should 
-  scale up and down
-  /home/aim/go-projects/cluster-api-actuator-pkg/src/github.com/openshift/cluster-api-actuator-pkg/pkg/e2e/autoscaler/autoscaler.go:229
-•SSSSSSSSSSSSSS
-Ran 1 of 15 Specs in 0.000 seconds
-SUCCESS! -- 0 Passed | 0 Failed | 0 Pending | 14 Skipped
---- PASS: TestE2E (0.00s)
-PASS
-ok      github.com/openshift/cluster-api-actuator-pkg/pkg/e2e   0.037s
+$ ./hack/ci-integration.sh -focus "Autoscaler should" -v -dryRun
```

Comment:
Is this real output? Didn't think it ran parallel by default?

Reply:
it used to, but i took it out in #148, see [this commit](https://github.com/openshift/cluster-api-actuator-pkg/commit/ba0bf725211685eec27100d740da468cc7b0f2d7).

i probably created this before that change happened.